content/01-install/02b-install-wallet-Unix.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Install Wallet on Linux
+metaTitle: Install Wallet on Linux
+---
+
+This section outlines how to install the Mantis wallet on your Linux machine.
+
+Follow these instructions to install the Mantis wallet or go [here for instructions to install the Mantis client](/install/install-client).
+
+
+## Requirements
+
+These are the prerequisites for installing Mantis Wallet:
+* A Java Virtual Machine (JVM) with version 1.8.x.
+    To check the JVM version, use `java -version`.
+    To install Java, follow [these instructions](https://java.com/en/download/help/download_options.html).
+    (Note that  the Mantis team has not tested the wallet with JVM 1.9 or later versions)
+* 4G of memory (RAM)
+    Additional RAM is needed for the DAG file _if mining_ is enabled.
+* 250GB of disk space to ensure fast sync of the node. You will need more space than that for the future growth of the chain. An SSD of at least 500G is recommended for full sync.
+
+## Install
+
+To install Mantis Wallet, follow these steps:
+
+1. Download the wallet binary from the Wallet page at [MantisClient.io](https://mantisclient.io/):
+![Mantis Wallet page](../images/1-Download.png).
+Choose and download the correct installer from the Wallet quick-start page:
+![Linux Installer](../images/2b-Download-Installer.png)
+Choose the Linux installer, and note the checksum.
+```
+Mantis-Wallet-<version>.AppImage
+Mantis-Wallet-<version>.dmg
+Mantis-Wallet.Setup.<version>.exe
+```
+
+> Remember to run checksum verification on your downloads. Refer to [this section](first-steps/downloads#checksum) for instructions.
+
+2. Move the binary file to your preferred folder and unzip the client archive file. Finally, remove the zip file.
+
+```
```

Comment:
I believe AppImage is what is inside the .zip file
and you can run the AppImage also from the command line
make sure the file is runnable "chmod u+x Mantis-Wall...AppImage"
then, after cd just "./Mantis-Wall...AppImage"